SEP-0002-revised.md

```diff
@@ -0,0 +1,248 @@
+
+Nominations, discussions and any elections of Project Members should take place before the initial election of the Steering Committee.
+
+
+### Active and Emeritus Status
+
+Project Members have no term limits.
+Project Members who are no longer actively contributing to the project should contact the Steering Committee to move themselves to emeritus status.
+Active Project Members who have not made any significant contributions for two years, will be contacted by the Steering Committee and be asked to move themselves to emeritus status.
+The Steering Committee can change a Project Member's status to emeritus.
+Emeritus Project Members will continue to be listed on the website.
+Emeritus Project Members are not able to participate in votes.
+An emeritus member may request to be marked as active at any time if they feel they are making contributions.
+
+## Removing Project Members
+
+In exceptional circumstances, it may be necessary to remove someone from the Project Members against their will.
+A vote must be held to remove a Project Member.
+Such a vote is triggered by a motion made by an active Project Member, which must be seconded by an additional Project Member.
+The vote must conclude no more than three months after the motion is seconded.
+Removal requires approval by two-thirds of all active Project Members at the time the motion is made.
+The motion, second, and vote will be by secret ballot.
+Removal under this provision will be reflected by updating the list of Project Members.
+A member removed via this mechanism does not have emeritus status.
+
+It may be necessary for the Ombudsperson and the Steering Committee to remove a Project Member for violations of the Code of Conduct.
+In this case, the Steering Committee, Ombudsperson and Board will work together to make this decision.
+
+# Ombudsperson
+
+The Ombudsperson represents the interests of the Astropy community by providing an alternative point of contact for sensitive issues such as code of conduct violations and ethical concerns.
+Candidates for this project role are publicly nominated by the Board, after which at least two weeks for comment is allowed, and then the nominee must be confirmed by two-thirds of the active Project Members.
+
+The Ombudsperson has no term limit but can resign at any time, or be removed by the same process as removing a Project Member.
+In the period between removal/resignation and new appointment, the Steering Committee will temporarily take over the responsibilities of the Ombudsperson, should the need arise.
+
+# Code of Conduct
+
+The SunPy organization shall have an explicit code of conduct to state the norms of peer interactions, including inclusiveness and respect, and describe the process for dispute resolution.
+The code of conduct shall be publicly posted on the SunPy website.
+The code of conduct shall be accessible in such a manner as to welcome and incorporate feedback from the developer community, even anonymous feedback.
```

Comment:
Do you know how the anonymous feedback will work?

Reply:
I think that system would be email the ombusperson, but beyond confidential@sunpy.org we don't have anything at the moment.

Reply:
I guess my question is, is that enough or do we need to improve on it and should it be defined here?

Reply:
I don't think this is a good place to define the processes, as they may want to be changed in a more flexible way than modifying this document, which will need a formal vote of the project members.